roles/example-cnf-app/tasks/app.yaml

```diff
@@ -11,30 +11,19 @@
   poll: 2
   failed_when: "net_list|length == 0"
 
-- name: Get the clusterversion info
```

Comment:
Can you add a when condition so that this step can be executed when one of ocp_version_maj or ocp_version_min is not defined. The intention of this repo is to be used as standalone usage too. Eventhough we can use all the params from CI, if not available, those should be fetched here.

Reply:
Thanks @krsacme, after talking to the team it was clear to me that while this could be tweaked it should remain independent from the CI. I'm brainstorming a different approach.